packages/formation-react/stories/sidenav.stories.mdx

```diff
@@ -116,7 +116,7 @@ Hierarchical, vertical navigation to place at the side of a page.
 * If your content is within a frame or sub-area of a page, consider ways to simplify your navigation system.
 
 ### Guidance
-* Typically used in a [two-column layout]({{ site.baseurl }}/layout/page-templates#two-columns-content-on-right).
+* Typically used in a [two-column layout](/layout/page-templates#two-columns-content-on-right).
```

Comment:
As-is, the `href` will be `/?path=/layout/page-templates#two-columns-content-on-right`. The `..` in front of the leading `/` will get it out of "storybook mode" so that we are changing the full route instead of just the `path` query param.

```suggestion
* Typically used in a [two-column layout](../layout/page-templates#two-columns-content-on-right).
```

<br />
<br />

In "normal mode":

![image](https://user-images.githubusercontent.com/2008881/100921052-dfd2c800-3490-11eb-9d1c-a7cb0ecd6fdb.png)

Reply:
Ugh. Confirmed.
![image](https://user-images.githubusercontent.com/12970166/101065536-1f111f80-354a-11eb-9014-96ae08607e26.png)

That's _so_ dumb.
![image](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fmedia.tenor.com%2Fimages%2F54fc394817457e4b96ef4669641375d9%2Ftenor.gif&f=1&nofb=1)